src/AzureIoTHub.Portal/Server/Controllers/v1.0/DeviceTagSettingsController.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) CGI France. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace AzureIoTHub.Portal.Server.Controllers.v10
+{
+    using Azure;
+    using Azure.Data.Tables;
+    using AzureIoTHub.Portal.Server.Factories;
+    using AzureIoTHub.Portal.Server.Mappers;
+    using AzureIoTHub.Portal.Shared.Models.v10.Device;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Logging;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+
+    [Route("api/[controller]")]
```

Comment:
Please change to route to be sure to support the correct URI :

```suggestion
    [Route("/api/settings/device-tags")]
```